test/multiarch_utils.go

```diff
@@ -59,6 +61,33 @@ func initImageNames() map[int]string {
 	}
 }
 
+// generate the map ready to search and replace image names with regexp for examples files.
+// search is done using "image: <name>" pattern.
+func GetImagesMappingRE() map[*regexp.Regexp][]byte {
```

Comment:
Does this method need to be exported?

Reply:
otherwise golangci-lint fails with
```
# make golangci-lint
🐱 running golangci-lint…
test/multiarch_utils.go:66:6: `getImagesMappingRE` is unused (deadcode)
func getImagesMappingRE() map[*regexp.Regexp][]byte {
     ^
Makefile:171: recipe for target 'golangci-lint' failed
make: *** [golangci-lint] Error 1
```
Will be happy to get workaround here

Reply:
Ughhhh, the tool is just wrong in this case, it's used by a var.

As a workaround, we could try:

```go
var imagesMappingRE map[*regexp.Regexp][]byte

func init() {
  imagesMappingRE =  getImagesMappingRE()
}
```

Reply:
we can also use the `// nolint` comments :wink:

Reply:
thanks @ImJasonH , with this modification golangci-lint checks passed